step_exec_lib/utils/processes.py

```diff
@@ -23,16 +23,18 @@ def run_and_handle_error(args: List[str], expected_error_text: str, **kwargs: An
         kwargs["text"] = True
 
     try:
-        run_res = subprocess.run(args, **kwargs, check=True, stderr=subprocess.PIPE)  # nosec
+        run_res = subprocess.run(args, **kwargs, check=True, stdout=subprocess.PIPE, stderr=subprocess.STDOUT)  # nosec
```

Comment:
This combines stderr and stdout. As both are useful when seeing the test results.